kokoro/linux/build.sh

```diff
@@ -123,6 +125,9 @@ test "${APP_EXIT_STATUS}" -eq 130
 
 # TODO(https://github.com/google/gapid/issues/3163): The coherent memory
 #  tracker must be disabled with SwiftShader for now.
-xvfb-run -e xvfb.log -a bazel-bin/pkg/gapit trace -device host -disable-coherentmemorytracker -disable-pcs -disable-unknown-extensions -record-errors -no-buffer -api vulkan -start-at-frame 5 -capture-frames 10 -observe-frames 1 -out out/dist/vulkan_sample.gfxtrace bazel-bin/cmd/vulkan_sample/vulkan_sample
+xvfb-run -e xvfb.log -a bazel-bin/pkg/gapit trace -device host -disable-coherentmemorytracker -disable-pcs -disable-unknown-extensions -record-errors -no-buffer -api vulkan -start-at-frame 5 -capture-frames 10 -observe-frames 1 -out $ARTIFACT_DIR/vulkan_sample.gfxtrace bazel-bin/cmd/vulkan_sample/vulkan_sample
 
-xvfb-run -e xvfb.log -a bazel-bin/pkg/gapit video -gapir-nofallback -type sxs -frames-minimum 10 -out vulkan_sample.mp4  out/dist/vulkan_sample.gfxtrace
+sync
```

Comment:
why? this looks like grasping at straws.

Reply:
removed